modules/reitit-ring/src/reitit/ring.cljc

```diff
@@ -28,11 +28,11 @@
               (update acc method expand opts)
               acc)) data http-methods)])
 
-(defn compile-result [[path data] {::keys [default-options-handler] :as opts}]
+(defn compile-result [[path data] {::keys [default-options-resource] :as opts}]
   (let [[top childs] (group-keys data)
         childs (cond-> childs
-                       (and (not (:options childs)) (not (:handler top)) default-options-handler)
-                       (assoc :options {:no-doc true, :handler default-options-handler}))
+                       (and (not (:options childs)) (not (:handler top)) default-options-resource)
```

Comment:
This introduces a silent breaking change. There should be an assertion for the old option to fail fast. We could remote the assertion with 1.0.0